crates/wallet/tests/wallet.rs

```diff
@@ -1392,6 +1392,60 @@ fn test_create_tx_global_xpubs_with_origin() {
     assert_eq!(psbt.xpub.get(&key), Some(&(fingerprint, path)));
 }
 
+#[test]
+fn test_create_tx_increment_change_index() {
+    use coin_selection::Error;
+    let (desc, change_desc) = get_test_tr_single_sig_xprv_with_change_desc();
```

Comment:
I guess you could have a test that tests for a single descriptor wallet, without the change descriptor. WDYT ?

Reply:
Let me know if you like the test better in 849e13691031cb811567775c54325ae51e67a901

Reply:
It looks good, thanks!